mlir/test/lib/Dialect/Test/TestTypeDefs.td

```diff
@@ -398,4 +398,8 @@ def TestTypeVerification : Test_Type<"TestTypeVerification"> {
   let assemblyFormat = "`<` $param `>`";
 }
 
+def TestTypeOpAsmTypeInterface : Test_Type<"TestTypeOpAsmTypeInterface", [DeclareTypeInterfaceMethods<OpAsmTypeInterface, ["getAsmName"]>]> {
```

Comment:
Nit: break the line to avoid it going too long

```
def TestTypeOpAsmTypeInterface : Test_Type<"TestTypeOpAsmTypeInterface",
   [DeclareTypeInterfaceMethods<OpAsmTypeInterface, ["getAsmName"]>]> {